src/Desktop/ArcGISRuntime.Desktop.Samples/Samples/Map/MapFromPortal/MapFromPortal.xaml.cs

```diff
@@ -0,0 +1,97 @@
+//you may not use this file except in compliance with the License.
+//You may obtain a copy of the License at
+//
+//http://www.apache.org/licenses/LICENSE-2.0
+//
+//Unless required by applicable law or agreed to in writing, software
+//distributed under the License is distributed on an "AS IS" BASIS,
+//WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//See the License for the specific language governing permissions and
+//limitations under the License.
+using Esri.ArcGISRuntime.Controls;
+using Esri.ArcGISRuntime.Portal;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Windows;
+
+
+namespace ArcGISRuntime.Desktop.Samples.MapFromPortal
+{
+    public partial class MapFromPortal
+    {
+        private ArcGISPortal _portal;
+
+        //Construct Load Map sample control
+        public MapFromPortal()
+        {
+            InitializeComponent();
+
+            Loaded += LoadMap_Loaded;
+        }
+
+        // Loads UI elements and an initial Map
+        private async void LoadMap_Loaded(object sender, RoutedEventArgs e)
+        {
+            try
+            {
+                _portal = await ArcGISPortal.CreateAsync();
+                var searchParams = new SearchParameters("type: \"web map\" NOT \"web mapping application\"");
+                var result = await _portal.ArcGISPortalInfo.SearchHomePageFeaturedContentAsync(searchParams);
+                comboMap.ItemsSource = result.Results;
+
+                var map = result.Results.FirstOrDefault();
+                if (map != null)
+                {
+                    comboMap.SelectedIndex = 0;
+                    await LoadMapAsync(map.Id);
+                }
+            }
+            catch (Exception ex)
+            {
+                detailsPanel.Visibility = Visibility.Visible;
+                MessageBox.Show(ex.Message, "Sample Error");
+            }
+        }
+
+        // Loads a webmap on load button click
+        private async void LoadButton_Click(object sender, RoutedEventArgs e)
+        {
+            string id = string.Empty;
+            if (comboMap.SelectedIndex >= 0)
+                id = comboMap.SelectedValue as string;
+            else
+                id = comboMap.Text;
+
+            await LoadMapAsync(id);
+        }
+
+        // Loads the given webmap
+        private async Task LoadMapAsync(string mapId)
+        {
+            try
+            {
+                progress.Visibility = Visibility.Visible;
+
+                var item = await ArcGISPortalItem.CreateAsync(_portal, mapId);
+                var map = new Esri.ArcGISRuntime.Map(item);
+                MyMapView.Map = map;
+ 
+                detailsPanel.DataContext = item;
+                detailsPanel.Visibility = Visibility.Visible;
+            }
+            catch (Exception ex)
+            {
+                detailsPanel.Visibility = Visibility.Visible;
+                MessageBox.Show(ex.Message, "Sample Error");
```

Comment:
Maybe give a bit more helpful error message to the ui then "Error code '400' : 'Item does not exist or is inaccessible.'".What do you think?               

could be something like
var errorMessage = $"Map cannot be loaded from a portal. \n{ex.Message}";
MessageBox.Show(errorMessage, "Sample error");